### PR TITLE
Add BMBLS format

### PR DESCRIPTION
**Pull request by Antidote**
Add support for parsing bmbls

### CODE REVIEW

File: mercury_engine_data_structures/formats/game_model_root.py [hunk 1]
```diff
@@ -4,14 +4,14 @@
 from mercury_engine_data_structures.formats.property_enum import PropertyEnum
 
 
-def create(name: str, magic_number: int):
+def create(name: str, magic_number: int, root=gameeditor_CGameModelRoot):
     return construct.Struct(
-        _magic=construct.Const(name, PropertyEnum),
-        _magic_number=construct.Const(magic_number, construct.Hex(construct.Int32ul)),
+        magic=construct.Const(name, PropertyEnum),
+        magic_number=construct.Const(magic_number, construct.Hex(construct.Int32ul)),
```
Comment (duncathan) [inline mercury_engine_data_structures/formats/game_model_root.py:10]: sorry I was looking at an old branch, you can revert this

File: mercury_engine_data_structures/formats/__init__.py [hunk 2]
```diff
@@ -14,6 +15,7 @@
 
 ALL_FORMATS = {
     "PKG": Pkg,
+    "BMBLS": bmbls,
```
Comment (duncathan) [inline mercury_engine_data_structures/formats/__init__.py:18]: `Bmbls`

**Review by duncathan — CHANGES_REQUESTED** [review]

File: mercury_engine_data_structures/formats/game_model_root.py (hunk 1 above)
Comment (Antidote) [reply mercury_engine_data_structures/formats/game_model_root.py]: Fixed

File: mercury_engine_data_structures/formats/__init__.py (hunk 2 above)
Comment (Antidote) [reply mercury_engine_data_structures/formats/__init__.py]: Fixed

**Review by duncathan — APPROVED** [review]

**Pull request merged**